antsibull/utils/transformations.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
```

Comment:
I'm hesitant on calling this a file for general transformations with only one example function (which is targeted specifically at splitting FQCNs into parts and then returning a mapping).  I'm also thinking that this and get_pkg_data.py might be better outside of the utils directory.

This is somewhat bikeshedding, though, so I guess it's just a heads up that I'll probably move and rename at some point in the future.